### ISSUE TITLE
KNNC-1 Implement auxiliary functions 

### ISSUE THREAD

**Issue by ILIIH** [ILIIH/K-neariest-neighbor-classifier#1]
Implement the function `compute_distances_two_loops`

The function build an array of distances between all vectors in the test set and in the training set.

As a result, they should build an array of size `(num_test, num_train)`, where the coordinate `[i][j]` corresponds to the distance between the i-th vector in test (`test[i]`) and the j-th vector in train (`train[j]`).q, We use Manhattan distance as metric.
